test/python/dockerpy/README.md

```diff
@@ -0,0 +1,40 @@
+# Dockerpy regression test
+
+Python test suite to validate Podman endpoints using dockerpy library
+
+## Running Tests
+
+To run the tests locally in your sandbox (Fedora 32):
+
+```shell script
+# dnf install python3-docker
+```
+
+### Run the entire test suite
+
+```shell
+# cd test/python/dockerpy
+# PYTHONPATH=/usr/bin/python python -m unittest discover .
```

Comment:
Is this (`PYTHONPATH`) a holdover from an earlier misunderstanding in docs? Unless I'm missing something profound, none of these `PYTHONPATHs` should be necessary, and they're really misleading.

If I am missing something, could you please document it?

Also, could I suggest using a `$` prompt instead of hash, to make it obvious to a user that this does not (and should not) need root?